CONTRIBUTING.md

```diff
@@ -8,14 +8,14 @@ Thank you for being interested in helping with Monolint!
 
 * Fork this project to your account
 
-- Implement it, create unit tests and run `make rules-doc` to update documentation
+- Implement it, create tests and run `make rules-doc` to update documentation
```

Comment:
The texts should be still referencing "unit tests", only the Makefile target should be changed